chill.sh

```
@@ -0,0 +1,35 @@
+
+prefix=$1
+chill_days=7
+
+if [ -z "${GITHUB_OUTPUT+x}" ] || [ -z "$GITHUB_OUTPUT" ]; then
+  GITHUB_OUTPUT=/dev/stdout
+fi
+
+scratch=$(mktemp -d -t tmp.XXXXXXXXXX)
+function finish {
+  rm -rf "$scratch"
+}
```

Shellcheck (0.11.0)

[warning] 'function' keyword is non-standard. Use 'foo()' instead of 'function foo'.

(SC2113)